apps/balancer-tools/src/app/apr/api/route.ts

```diff
@@ -10,6 +10,9 @@ import { filterPoolStats } from "./(utils)/filter";
 import { limitPoolStats, Order, sortPoolStats } from "./(utils)/sort";
 import { QueryParamsSchema } from "./(utils)/validate";
 
+export const dynamic = "force-dynamic";
```

Comment:
Not sure adding `"force-dynamic"` here does much. Did you take a look into the docs/notice any difference? I'd have added it to the root layout, not the here.

Reply:
I'm dropping this for now. The docs say that this will miss all the cache 🫠

Reply:
FYI just to make i clear this would miss all Vercel cache, not our own